framework/web_app/src/models/ScrapedProductOffer.ts

```diff
@@ -2,6 +2,8 @@ export type ScrapedProductOffer = {
     brand: string | null
     image: string
     is_available: boolean
+    is_saved: boolean
+    is_saved_product_active?: boolean
```

Comment:
Worth making it nullable? If false and null result in the same behaviour I would just make it simpler